tests/queries/0_stateless/02715_exponential_smoothing_alpha.sql

```diff
@@ -0,0 +1,42 @@
+/* exponentialSmoothingAlpha tests */
+SELECT exponentialSmoothingAlpha()(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0.5, 0.5)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(-1)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7 as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(2)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0.5)(value, timestamp) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0.5) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99; -- { serverError BAD_ARGUMENTS }
+SELECT exponentialSmoothingAlpha(0)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
+SELECT exponentialSmoothingAlpha(0.2)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 2 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
+SELECT exponentialSmoothingAlpha(0.5)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 3 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
+SELECT exponentialSmoothingAlpha(0.8)(value) over (ORDER BY timestamp ASC) from (SELECT number as timestamp, number * 4 - (number % 7 - 3) / 7  as value from numbers_mt(100)) OFFSET 99;
```

Comment:
I would find one or two tests with more "advanced" window function syntax interesting: `PARTITION BY` clause, and a limited frame `Rows BETWEEN ... PRECEDING AND CURRENT ROW`) - just to check that it works as expected.